packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts

```diff
@@ -262,4 +262,7 @@ export interface IGroupDividerProps {
 
   /** Override which allows the caller to provider a custom title. */
   onRenderTitle?: IRenderFunction<IGroupDividerProps>;
+
+  /** Props for expand/collapse button */
+  expandCollapseButtonProps?: React.HTMLAttributes<HTMLButtonElement>;
```

Comment:
it appears that we're calling this the "showAll" , right? If that's correct, can we use that as the type name?